### ISSUE TITLE
compressed slice files missing from Load/Unload menu

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#188]
```
Slice files are missing from the Load/Unload menu when slice files are 
compressed then deleted (leaving just the compressed slice files)

```

Original issue reported on code.google.com by `gforney` on 2007-10-19 00:17:29

**Comment by gforney** [comment]
```
This bug is fixed.  I'm posting this here so that we will have a record of this bug

in case someone reports it later (for Smokeview's 5.0.2 or earlier).  It will be 
fixed in the soon to be released Smokeview 5.0.3
```

Original issue reported on code.google.com by `gforney` on 2007-10-19 00:18:52

**Comment by gforney** [comment]
```
presume verified
```

Original issue reported on code.google.com by `gforney` on 2007-10-24 01:34:34
